### PR TITLE
Review api endpoints

### PR DESCRIPTION
**Pull request by fleko**

### CODE REVIEW

**Comment by ignaciojonas** [comment]
Looks good to me.

File: lib/auth0/api/v2/users.rb [hunk 1]
```diff
@@ -140,15 +140,38 @@ def link_user_account(user_id, body)
...
+          raise Auth0::MissingUserId, 'Must supply a valid secondary user_id' if secondary_user_id.to_s.empty?
+          raise Auth0::InvalidParameter, 'Must supply a valid provider' if provider.to_s.empty?
           path = "#{users_path}/#{user_id}/identities/#{provider}/#{secondary_user_id}"
           delete(path)
         end
 
+        # Retrieve every log event for a specific user id
+        # @see https://auth0.com/docs/api/management/v2#!/Users/get_logs_by_user
+        # @param user_id [string] The user_id of the logs to retrieve.
+        # @param per_page [integer] The amount of entries per page. Default: 50. Max value: 100.
```
Comment (falvarisqueta) [inline lib/auth0/api/v2/users.rb:155]: do we need a test for max value?

File: lib/auth0/api/v2/users.rb [hunk 2]
```diff
@@ -140,15 +140,38 @@ def link_user_account(user_id, body)
...
+        # @param page [integer]  The page number. Zero based.
+        # @param include_totals [boolean] True if a query summary must be included in the result.
+        # @param sort [string] The field to use for sorting. 1 == ascending and -1 == descending.
```
Comment (falvarisqueta) [inline lib/auth0/api/v2/users.rb:158]: what happens if sorting is != than 1 or -1?

**Comment by falvarisqueta** [comment]
Looks good to me

**Pull request merged**